presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -87,7 +87,7 @@ public AccessControlManager(TransactionManager transactionManager)
         addSystemAccessControlFactory(new FileBasedSystemAccessControl.Factory());
     }
 
-    public void addSystemAccessControlFactory(SystemAccessControlFactory accessControlFactory)
+    public final void addSystemAccessControlFactory(SystemAccessControlFactory accessControlFactory)
```

Comment:
Why make this particular method final?

Reply:
It fixes the warning that an overridable method is used in the constructor (should be safe in this case but it's best practice).